draft-ietf-dnsop-svcb-https.md

```diff
@@ -127,7 +127,8 @@ Additional goals specific to HTTPS RRs and the HTTP use-cases include:
 * Enable SRV-like benefits (e.g. apex delegation, as mentioned above) for HTTP,
   where SRV {{?SRV=RFC2782}} has not been widely adopted
 * Provide an HSTS-like indication {{!HSTS=RFC6797}} signaling that the "https"
-  scheme should be used instead of "http" for this request (see {{hsts}}).
+  scheme should be used instead of "http" for all requests to this origin (see
+  {{hsts}}).
```

Comment:
An unqualified "origin" I would tend to treat as meaning "URI origin", but that is scheme-specific.  {{HTTP}} defines an "origin server" that may be closer to what we mean here.

Reply:
"Origin server" doesn't seem to quite capture this either.  Changed to "host and port".